openrtb/openrtbtest/testutil.go

```diff
@@ -152,7 +152,9 @@ func NewBidRequestForTesting(id string, impId string) *openrtb.BidRequest {
 // NewBidRequestWithFloorPriceForTesting method creates a bid request with a price in USD.
 func NewBidRequestWithFloorPriceForTesting(id string, impId string, price float64) *openrtb.BidRequest {
 	br := NewBidRequestForTesting(id, impId)
-	br.Impressions[0].BidFloorPrice = price
+	var new_price float64
```

Comment:
new_price -> `newPrice`, always use camel case for variable names.

Also assignment and value in one line:
```go
newPrice := price
```

Now, nitpick on the names, I think what you really want to do is to create a pointer value for this, maybe you can call this variable `priceForRef`?